modules/browser/BrowserCapabilities.js

```diff
@@ -241,11 +248,40 @@ export default class BrowserCapabilities extends BrowserDetection {
      * @returns {boolean}
      */
     usesNewGumFlow() {
-        return (this.isChrome()
-                && !this.isVersionLessThan('61'))
-            || this.isFirefox()
-            || this.isSafariWithWebrtc();
+        const REQUIRED_CHROME_VERSION = 61;
+
+        if (this.isChrome()) {
+            return !this.isVersionLessThan(REQUIRED_CHROME_VERSION);
+        }
+
+        if (this.isFirefox() || this.isSafariWithWebrtc()) {
+            return true;
+        }
+
+        if (this.isNWJS()) {
```

Comment:
why is this needed as a separate case from isChromeLike?  Is it because NWJS represents the chrome version differently than others?  If so, could that be handled somewhere else so it remains more uniform with the rest of the isChromeLike tests?

Reply:
if it is just the version thing, i wonder if it might be clearer to still have this as a case within the `if (this.isChromeLike()) {` block below?

Reply:
>  Is it because NWJS represents the chrome version differently than others?

Yep, it's not in `navigator.userAgent` but in `process`.

Reply:
I don't think the extra indentation level adds much IMHO. That's why I left it separate. I'll give it a shot when I rename the function and see how it feels like.